contributing_to_docs/doc_guidelines.adoc

```diff
@@ -410,13 +410,18 @@ Follow these general guidelines when using code blocks:
 
 * Try to use callouts to provide information on what the output represents when required.
 
-For all code blocks, you must include an empty line above a code block.
+For all code blocks, you must:
```

Comment:
Do we want to make this sound like a hard requirement, or more just that we are now saying explicitly it's OK to use if you want?

Reply:
Thanks @adellape I'll update the language.